cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -149,6 +150,9 @@ public BuildCommand(Path userDir, PrintStream outStream, PrintStream errStream,
     @CommandLine.Option(names = "--dump-bir", hidden = true)
     private boolean dumpBIR;
 
+    @CommandLine.Option(names = "--dump-bir-file", hidden = true)
```

Comment:
How about --dump-bir-binary?

Reply:
@sameerajayasoma , looks like with the option `--dump-bir-file` you have to give output file name/location. it is not a boolean flag apparently